auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java

```diff
@@ -386,7 +383,7 @@ private void showLoadingDialog(String message) {
 
     private void dismissLoadingDialog() {
         if (mProgressDialog != null) {
-            mProgressDialog.dismiss();
+            mProgressDialog.dismissAllowingStateLoss();
```

Comment:
You work on Santa Tracker too??? Dang, you're all over the place! 😆

At least you know now 😄